### ISSUE TITLE
Scratch blocked replit!

### ISSUE THREAD

**Issue by RitewayYT** [TimMcCool/scratchattach#123]
Can you use a proxy instead?

**Comment by programORdie2** [comment]
Yup

**Comment by TimMcCool** [comment]
maybe use a different online ide like codesandbox.io, or run it locally on your computer

**Comment by RC9910-Scratch** [comment]
Apparently using proxies in replit is against their ToS (#110), but as @TimMcCool said, you can use codesandbox.io

**Comment by TheCommCraft** [comment]
> Apparently using proxies in replit is against their ToS (#110), but as @TimMcCool said, you can use codesandbox.io

No, it isn't. It's not allowed to use replit to create a proxy. Using one is.

**Comment by RitewayYT** [comment]
Everyone thanks for the suggestions
